### ISSUE TITLE
Vue 3 'inline' boolean prop always has null value

### ISSUE THREAD

**Issue by jdevinemt** [iconify/iconify#201]
I'm using the web component version of Iconify and have created a custom Vue 3 component to wrap it. When doing so, I'm running into a weird issue where I am accepting an `inline` prop and binding it to the `inline` attribute of the `<iconify-icon>` element. However, whenever the value of the prop is `false` or `null`, the iconify element is always rendered as `<iconify-icon inline="null">`. According to the Vue documentation, it should be rendered as `inline="false"` when the prop has a value of `false`, and the `inline` attribute shouldn't be rendered at all when the prop value is `null`. I've verified that these expected results work on other html elements.

Here is my Vue component:
```vue
<template>
  <iconify-icon
      ref="iconElement"
      :class="$attrs.class"
      :icon="icon"
      :inline="inline"
  ></iconify-icon>
</template>

<script>
export default {
  inheritAttrs: false
}
</script>

<script setup>

const props = defineProps({
  icon: {
    type: String,
    required: true
  },
  inline: {
    type: Boolean,
    default: null // same issue occurs with a value of 'false'
  }
});

</script>
```

I've registered the component as 'app-icon' in my Vue app, here is what the component usage that produces this issue looks like:
```html
<iconify-icon icon="mdi:alert"></iconify-icon>
```
Which results in the following output when rendered:
```html
<iconify-icon class="" icon="mdi:alert" inline="null"></iconify-icon>
```

I have updated my Vue config to recognize the `iconify-icon` class as a custom html element. For the time being, I am just adding the attribute via the `onMounted` lifecycle hook based on the prop's value, but it's not an ideal situation since it occurs after the initial render.

Any idea what could be causing this? I'm not sure if it specific to Iconify or something to do web components in general.

**Comment by cyberalien** [comment]
Weird behaviour. Could be a web components issue in Vue 3.

All that attribute does is adds `vertical-align: -0.125em` to style, so instead of setting `inline` attribute, you can set style or add a class that sets that style.

**Comment by jdevinemt** [comment]
Actually, I think I found the cause. In the setter for `inline` the value is being set to `null` when the provided value is falsy. I believe it should instead be removing the attribute.

Current:
```javascript
set inline(value) {
     this.setAttribute('inline', value ? 'true' : null);
}
```

Proposed:
```javascript
set inline(value) {
   if(value){
     this.setAttribute('inline', value ? 'true' : null);
   }else{
     this.removeAttribute('inline');
   }
}
```

According to the [Mozilla web docs](https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute):
> You should use removeAttribute() instead of setting the attribute value to null either directly or using [setAttribute()](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute). Many attributes will not behave as expected if you set them to null.

Also, according to the same docs, there is no need to check whether the attribute is set or not before removing it.
> If the specified attribute does not exist, removeAttribute() returns without generating an error.

**Comment by cyberalien** [comment]
Doh! Thanks a lot!

**Comment by cyberalien** [comment]
I've published update with `dev` tag. Can you check if it fixes issue?

**Comment by jdevinemt** [comment]
Oops, you're so quick. I just made a pull request, but you can ignore it. I'll test your fix real quick.

**Comment by jdevinemt** [comment]
> I've published update with `dev` tag: `iconify-icon@dev`. Can you check if it fixes issue?

That did the trick. Thanks so much! I'll watch for when this gets added to the main release.

**Comment by cyberalien** [comment]
Published 1.0.4, included your pull request. Thanks a lot!
